spec/merchant_repository_spec.rb

```diff
@@ -211,4 +211,29 @@
       expect(mr.average_average_price_per_merchant).to eq 3.55
     end
   end
+
+  describe '#merchants_with_only_one_item' do
+    it 'returns a collection of merchanta who only have one item' do
+      engine = double('engine')
+      items = double('item_repo')
+      allow(mr).to receive(:engine).and_return(engine)
+      allow(engine).to receive(:items).and_return(items)
+      allow(items).to receive(:item_count).and_return(['item1'])
+
```

Comment:
I am not sure these mocks are necessary, we need to stub the Merchants response up the tree, when it calls item.count. This will still work though because the Merchant will send a message to 'mr' that will ask for the 'engine', etc